Singer.API/ClientApp/src/app/modules/legalguardians/services/createCareRegistration.dto.ts

```diff
@@ -0,0 +1,7 @@
+import { EventRegistrationTypes } from '../shared/register-care-wizard/register-care-wizard.component';
```

Comment:
What's this DTO doing here? It looks like it lost its mommy, please explain it how to find its way to the DTOs folder ;)